integration/mediation-tests/tests-other/src/test/java/org/wso2/carbon/esb/synapse/common/formatters/ESBJAVA3290TestXWWWFormURLEncodedFormatter.java

```diff
@@ -82,7 +70,7 @@ public void testPostRequest() throws Exception {
     public void stop() throws Exception {
         cleanup();
         Thread.sleep(3000);
-        serverConfigurationManager.restoreToLastConfiguration();
-        serverConfigurationManager = null;
+//        serverConfigurationManager.restoreToLastConfiguration();
```

Comment:
Better to remove commented code if no usage

Reply:
fixed